manager/test/api/src/test/resources/test-plans/searching-testPlan.xml

```diff
@@ -38,9 +66,4 @@
     <test name="Invalid Filter Operator">test-plan-data/searching/validation/006_query-invalid-filter-operator.resttest</test>
   </testGroup>
 
-  <!-- How about some user testing? -->

```

Comment:
Why do we delete this?

Reply:
@BastianGem this one is for you. I guess you felt there was enough user testing?

Reply:
I just moved this one to Misc lines 53-56